src/main/webapp/resources/js/ardc2.js

```diff
@@ -0,0 +1,324 @@
+var ardcDisplaySelector = "span[data-cvoc-protocol='ardc']";
+var ardcInputSelector = "input[data-cvoc-protocol='ardc']";
+
+$(document).ready(function() {
+    getData().then((data) => {
+        expandARDC(data);
+        updateARDCInputs(data);
+    });
+});
+
+function getData() {
+    return new Promise((resolve, reject) => {
+    	//Until CORS is supported for this URI, get a local copy at a hardcoded URL
+        //          fetch("https://dataverse-tools.ada.edu.au/api/forCode")
+        fetch("http://ec2-3-238-111-188.compute-1.amazonaws.com/resources/js/forCode")
+        //That URI gives a 201 response but the local URL just sends 200
+            //.then((dd) => dd.status === 201 && dd.json())
+            .then((dd) => dd.status === 200 && dd.json())
+            .then((data) => resolve(data))
+            .catch((err) => console.log(err));
+    });
+}
+
+function expandARDC(data) {
+    // Check each selected element
+    $(ardcDisplaySelector).each(function() {
+        var ardcDisplayElement = this;
+        // If it hasn't already been processed
+        if (!$(ardcDisplayElement).hasClass('expanded')) {
+            // Mark it as processed
+            $(ardcDisplayElement).addClass('expanded');
+            //Retrieve parameters from the element's attributes
+            //The service URL to contact using this protocol
+            var cvocUrl = $(ardcDisplayElement).attr('data-cvoc-service-url');
+            //The language requested
+            var lang = ardcDisplayElement.hasAttribute("lang") ? $(ardcDisplayElement).attr('lang') : "";
+            //The value in the element. Currently, this must be either the URI of a term or plain text - with the latter not being formatted at all by this script
+            var id = ardcDisplayElement.textContent;
+            let names = findTerm(id, data);
+            console.log(JSON.stringify(names));
```

Comment:
Do we want all these console.log's across the new Javascript that's being added?

Reply:
This script is the one that is still proof-of-concept/in development, so the console.log statements are still useful. That said, the scripts are now in the other repo and I'll be shifting this to a dev branch so it is clearer that this one is not yet ready to go.